web/migrations/0061_alter_educationalvideo_description_videorequest.py

```
@@ -0,0 +1,76 @@
+                            ("approved", "Approved"),
+                            ("fulfilled", "Fulfilled"),
+                            ("rejected", "Rejected"),
+                        ],
+                        default="pending",
+                        max_length=20,
+                    ),
+                ),
+                (
+                    "category",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.PROTECT, related_name="video_requests", to="web.subject"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)